packages/design-system/src/components/Alert/Alert.tsx

```diff
@@ -195,6 +216,7 @@ export class Alert extends React.PureComponent<
         aria-labelledby={heading ? this.headingId : undefined}
         {...alertProps}
       >
+        {this.getIcon()}
```

Comment:
Alert size changed slightly but not position. Seems fine to me.

![alerts](https://user-images.githubusercontent.com/7595652/147166391-d058713f-42d6-4a55-a8bf-720435cdb98a.gif)